<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowCSSH</class>
 <widget class="QMainWindow" name="MainWindowCSSH">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cluster Secure Shell</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="commandLineEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="executePushButton">
          <property name="text">
           <string>Execute</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QMdiArea" name="mdiArea"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuConnection">
    <property name="title">
     <string>Connection</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionClose"/>
    <addaction name="actionClose_all"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>?</string>
    </property>
    <widget class="QMenu" name="menuLicence">
     <property name="title">
      <string>Licence</string>
     </property>
     <addaction name="actionComplete"/>
     <addaction name="actionWarranty"/>
     <addaction name="actionCopy"/>
    </widget>
    <addaction name="actionHelp"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
    <addaction name="menuLicence"/>
    <addaction name="actionAbout_Qt"/>
    <addaction name="actionShow_network_information"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuEdit_config_file">
     <property name="title">
      <string>Edit config file</string>
     </property>
     <addaction name="actionMain_config_file"/>
     <addaction name="actionOther"/>
    </widget>
    <addaction name="actionLoad_script"/>
    <addaction name="menuEdit_config_file"/>
   </widget>
   <addaction name="menuConnection"/>
   <addaction name="menuFile"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew">
   <property name="text">
    <string>Open connection to servers</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionClose_all">
   <property name="text">
    <string>Close all</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+T</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+Q</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionComplete">
   <property name="text">
    <string>Complete</string>
   </property>
  </action>
  <action name="actionWarranty">
   <property name="text">
    <string>Warranty</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionLoad_script">
   <property name="text">
    <string>Load script</string>
   </property>
  </action>
  <action name="actionMain_config_file">
   <property name="text">
    <string>Main config file</string>
   </property>
  </action>
  <action name="actionOther">
   <property name="text">
    <string>Other</string>
   </property>
  </action>
  <action name="actionShow_network_information">
   <property name="text">
    <string>Show network information</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+N</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
